pkg/helpers/tokencmd/request_token_test.go

```diff
@@ -1,745 +0,0 @@
-package tokencmd
-
-import (
-	"bytes"
-	"encoding/json"
-	"encoding/pem"
-	"errors"
-	"fmt"
-	"net/http"
-	"net/http/httptest"
-	"reflect"
-	"testing"
-
-	"github.com/RangelReale/osincli"
-
-	"k8s.io/apimachinery/pkg/util/diff"
-	restclient "k8s.io/client-go/rest"
-
-	"github.com/openshift/library-go/pkg/oauth/oauthdiscovery"
-)
-
-type unloadableNegotiator struct {
```

Comment:
These negotiator tests seem useful and maybe we should keep them under authchallengers in negotiate_tests.go. WDYT?

Reply:
good catch, I'll see where would be a good place for them

Reply:
the tests are now back :slightly_smiling_face: